api/src/pcapi/connectors/api_allocine.py

```diff
@@ -51,6 +52,42 @@ def get_movie_list_page(after: str = "") -> allocine_serializers.AllocineMovieLi
     return validated_response
 
 
+def filter_invalid_edge(json_data: dict, locs: tuple) -> dict:
+    if not locs or set(locs[:2]) != {"movieShowtimeList", "edges"}:
+        # nothing to do (to complicated or locs is malformed), let it fail
+        return json_data
+
+    edge_index = locs[2]
+    invalid_edge = copy.deepcopy(json_data["movieShowtimeList"]["edges"][edge_index])
+
+    enumerate_edges = enumerate(json_data["movieShowtimeList"]["edges"])
+    json_data["movieShowtimeList"]["edges"] = [edge for idx, edge in enumerate_edges if idx != edge_index]
```

Comment:
Je crois que l'on peut faire plus simple en faisant
```suggestion
    invalid_edge = json_data["movieShowtimeList"]["edges"].pop(locs[2])
```